### ISSUE TITLE
Vote: Release 0.0.6 (update: 0.5.0)

### ISSUE THREAD

**Issue by benalexau** [lmdbjava/lmdbjava#58]
It's been about 5 months since 0.0.5 was released, so there have been a number of dependency updates and some changes to `CursorIterator` to make it a little more efficient and flexible.

Are we fine to release 0.0.6?

**Comment by krisskross** [comment]
Go for it! But actually, I think the project feels a lot more mature than 0.0.6 in both stability and functionality.

Its only psychological and I don't mind 0.0.6 either.

**Comment by phraktle** [comment]
+1

Suggest bumping native to 0.9.21

**Comment by benalexau** [comment]
I agree re the version number, having used this in my own projects for over a year without a problem. Most APIs are package protected or final, so we've had no breaking backward compatibility changes to date and nor are they likely in the foreseeable future.

I've made the next version 0.5.0 so we're half way to 1.0.0. I think we should defer calling it 1.0.0 until we've tested with Java 9 GA and thus retain some flexibility if required. This is also why I didn't make it 0.9.21, which was a good suggestion by @phraktle but would mean we don't have any minor version numbers (ie major.minor.patch) remaining before 1.0.0.
